#Current Position
float32 x
float32 y
float32 theta

#Current Velocity
float32 speed
float32 angular
